lib/assets/javascripts/twine.coffee

```diff
@@ -219,6 +259,20 @@
 
   requiresRegistry = (args) -> /\$registry/.test(args)
 
+  nodeHasArrayIndexes = (node) ->
+    return unless node.bindingId?
+    !!elements[node.bindingId]?.indexes?
+
+  arrayPointersForNode = (node, context) ->
+    return {} unless node.bindingId?
+    indexes = elements[node.bindingId]?.indexes
+    return {} unless !!indexes?
```

Comment:
This is clever, but a pretty unclear. What do you think of making the "allow 0" aspect a little more explicit?

`return {} unless indexes || indexes == 0`